com.github.alexkdeveloper.radio.json

```diff
@@ -0,0 +1,42 @@
+{
+  "app-id": "io.github.alexkdeveloper.radio",
```

Comment:
the manifest filename no longer match the app-id.

also remember only the dbus name matching the app-id is granted ownership by default. And I suspect that the icons and desktop files will also be mis-named in that situation.